flake.nix

```diff
@@ -0,0 +1,69 @@
+{
+  description = "Apple M1 support for NixOS";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
```

Comment:
I would like to keep the same nixpkgs version that was in `pins.nix` and preserve the comment about the Hydra `mobile-nixos` jobset. This means users/CI won't have to wait forever to cross-compile the basic system.

It would break `nix flake update` though since that would be a no-op. Maybe it is better to document how to upgrade to a specific commit (I am not sure myself)?

Reply:
You can pin a specific commit with
```nix
    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable/[commit]";
```
or (I think)
```nix
    nixpkgs = {
      url = "github:nixos/nixpkgs/nixpkgs-unstable";
      rev = "[commit]";
    };
```